### PR TITLE
[CI] Complete support for APIScan in the CI

### PR DESCRIPTION
**Pull request by mandel-macaque**
At the moment even with issues the build will be successful so that we have green builds while we identify and fix several issues.

### CODE REVIEW

File: tools/devops/automation/templates/governance/apiscan.yml [hunk 1]
```diff
@@ -23,6 +23,55 @@ steps:
...
+    GdnPublishTsaExportedResultsPublishable: true
+  displayName: Create bugs for compliance tools results
+  condition: succeededOrFailed()
+  env:
+    WorkingDirectory: '$(Build.SourcesDirectory)/xamarin-macios' 
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
+  displayName: Break on compliance issues
+  inputs:
+    GdnBreakAllTools: false # do not breaj for APIScan until we have fixe the issues
```
Comment (dalexsoto) [inline tools/devops/automation/templates/governance/apiscan.yml:66]:
```suggestion
    GdnBreakAllTools: false # do not break for APIScan until we have fixe the issues
```

**Review by dalexsoto — APPROVED** [review]

File: tools/devops/automation/templates/governance/apiscan.yml [hunk 2]
```diff
@@ -23,6 +23,55 @@ steps:
...
+  displayName: Download artifacts
+  inputs:
+    artifactName: not-signed-package
+    allowFailedBuilds: true
+    path: $(Build.SourcesDirectory)/not-signed-package
+
+- pwsh: >-
+    ./prepare_workload_apiscan.ps1
+    -SourcesDirectory "$(Build.SourcesDirectory)"
+    -AndroidHash $Env:PLATFORM
```
Comment (rolfbjarne) [inline tools/devops/automation/templates/governance/apiscan.yml:36]: Android?

File: tools/devops/automation/templates/governance/apiscan.yml [hunk 3]
```diff
@@ -23,6 +23,55 @@ steps:
...
+    GdnPublishTsaExportedResultsPublishable: true
+  displayName: Create bugs for compliance tools results
+  condition: succeededOrFailed()
+  env:
+    WorkingDirectory: '$(Build.SourcesDirectory)/xamarin-macios' 
+
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
+  displayName: Break on compliance issues
+  inputs:
+    GdnBreakAllTools: false # do not break for APIScan until we have fixe the issues
```
Comment (rolfbjarne) [inline tools/devops/automation/templates/governance/apiscan.yml:66]:
```suggestion
    GdnBreakAllTools: false # do not break for APIScan until we have fixed the issues
```

**Review by rolfbjarne — APPROVED** [review]

File: tools/devops/automation/templates/governance/apiscan.yml (hunk 2 above)
Comment (mandel-macaque) [reply tools/devops/automation/templates/governance/apiscan.yml:36]: Manuel in the morning..

Comment (mandel-macaque) [inline tools/devops/automation/templates/governance/apiscan.yml:36]:
```suggestion
    -Platform $Env:PLATFORM
```

**Review by mcumming — APPROVED** [review]

**Review by haritha-mohan — APPROVED** [review]

**Pull request merged**